internal/cmdupdate/cmdupdate_test.go

```diff
@@ -169,14 +170,63 @@ func TestCmd_failUnCommitted(t *testing.T) {
 	updateCmd := cmdupdate.NewRunner(fake.CtxWithDefaultPrinter(), "kpt")
 	updateCmd.Command.SetArgs([]string{g.RepoName})
 	err = updateCmd.Command.Execute()
-	if !assert.Error(t, err) {
+	if !assert.NoError(t, err) {
```

Comment:
These assert statements have started to confuse me now :) generally we would expect these to terminate when assertion is not true. May be something to do with the assert library.